### PR TITLE
Support extract_method, extract_expression and change_visibility

### PR DESCRIPTION
**Pull request by alypeng**

### CODE REVIEW

File: phpactor.el [hunk 1]
```diff
@@ -787,9 +789,9 @@ function."
 
 ;;;###autoload
 (defun phpactor-extract-constant ()
-  "Execute Phpactor RPC extract-constant action."
+  "Execute Phpactor RPC extract_constant action."
```
Comment (kermorgant) [inline phpactor.el]: :+1:

File: phpactor.el [hunk 2]
```diff
@@ -787,9 +789,9 @@ function."
...
   (interactive)
-  (let ((arguments (phpactor--command-argments :source :offset :path)))
+  (let ((arguments (phpactor--command-argments :source :path :offset)))
```
Comment (kermorgant) [inline phpactor.el]: does that make a difference ?

File: phpactor.el [hunk 3]
```diff
@@ -799,5 +801,26 @@ function."
...
+  (let ((arguments (phpactor--command-argments :source :path :offset_start :offset_end)))
+    (apply #'phpactor-action-dispatch (phpactor--rpc "extract_expression" arguments))))
+
+;;;###autoload
+(defun phpactor-change-visibility ()
+  "Execute Phpactor RPC change_visibility action."
+  (interactive)
+  (let ((arguments (phpactor--command-argments :source :path :offset)))
+    (apply #'phpactor-action-dispatch (phpactor--rpc "change_visibility" arguments))))
+
```
Comment (kermorgant) [inline phpactor.el]: looks good to me, thanks :-)

File: phpactor.el (hunk 2 above)
Comment (alypeng) [reply phpactor.el]: I don't think so

It seems slightly more consistent to me because `:path` is used more frequently

**Comment by zonuexe** [comment]
@alypeng Thanks!

**Pull request merged**